src/utils/composeReducers.js

```diff
@@ -0,0 +1,12 @@
+import mapValues from '../utils/mapValues';
+import pick from '../utils/pick';
+
+export default function composeReducers(reducers) {
+  const finalReducers = pick(reducers, (val) => typeof val === 'function');
+
+  return function Composition(atom = {}, action) {
```

Comment:
@dbrans 

1. “Atom” is leftover from earlier versions of API. Need to be changed to `state`.
2. We want the default values for reducers to be `undefined`. For `state[key]` to be undefined initially, `state` needs to be `{}`. If it's `undefined` itself, `state[key]` will just crash.
3. You're right, we should lower case that.